mssw/Animal/lib/Horse.pm

```diff
@@ -0,0 +1,146 @@
+package Horse;
+
+use 5.006;
+use strict;
+use warnings;
+
+=head1 NAME
+
+Horse - The great new Horse!
+
+=head1 VERSION
+
+Version 0.01
+
+=cut
+
+our $VERSION = '0.01';
+
+use parent qw(Animal);
+
+=head1 SYNOPSIS
+
+Quick summary of what the module does.
+
+Perhaps a little code snippet.
+
+    use Horse;
+
+    my $foo = Horse->new();
+    ...
+
+=head1 EXPORT
+
+A list of functions that can be exported.  You can delete this section
+if you don't export anything, such as for a purely object-oriented module.
+
+=head1 SUBROUTINES/METHODS
+
+=head2 function1
+
+=cut
+
+sub function1 {
+}
+
+=head2 function2
+
+=cut
+
+sub function2 {
+}
```

Comment:
I actually didn't know they were there!

I couldn't see them through the pod...